frontend/src/concepts/pipelines/content/compareRuns/CompareRunTableRow.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { TableText, Td, Tr } from '@patternfly/react-table';
+import { Link } from 'react-router-dom';
+import { PipelineRunKFv2 } from '~/concepts/pipelines/kfTypes';
+import { CheckboxTd, TableRowTitleDescription } from '~/components/table';
+import {
+  RunCreated,
+  RunDuration,
+  RunStatus,
+} from '~/concepts/pipelines/content/tables/renderUtils';
+import useExperimentById from '~/concepts/pipelines/apiHooks/useExperimentById';
+import usePipelineRunVersionInfo from '~/concepts/pipelines/content/tables/usePipelineRunVersionInfo';
+import { PipelineVersionLink } from '~/concepts/pipelines/content/PipelineVersionLink';
+import { runDetailsRoute } from '~/routes';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { getPipelineJobExecutionCount } from '~/concepts/pipelines/content/tables/utils';
+
+type CompareRunTableRowProps = {
+  isChecked: boolean;
+  onToggleCheck: () => void;
+  run: PipelineRunKFv2;
+};
+
+const CompareRunTableRow: React.FC<CompareRunTableRowProps> = ({
+  isChecked,
+  onToggleCheck,
+  run,
+}) => {
+  const { namespace, getJobInformation } = usePipelinesAPI();
+  const [experiment] = useExperimentById(run.experiment_id);
+  const { version, loaded: isVersionLoaded, error: versionError } = usePipelineRunVersionInfo(run);
+  const { data: jobData } = getJobInformation(run.recurring_run_id);
+
+  return (
+    <Tr>
+      <CheckboxTd id={run.run_id} isChecked={isChecked} onToggle={onToggleCheck} />
+      <Td dataLabel="Run">
+        <TableRowTitleDescription
+          title={
+            <TableText wrapModifier="truncate">
+              <Link to={runDetailsRoute(namespace, run.run_id, run.experiment_id)}>
+                {run.display_name}
+              </Link>
+              {jobData &&
+                ` (Run ${getPipelineJobExecutionCount(run.display_name)} of ${
+                  jobData.display_name
```

Comment:
this should just be the description. i think Yan just had the Run ___ of ___ as a placeholder